cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -95,11 +95,11 @@ object ReporterCommand : CommandWithHelp() {
     private var resolutionsFile: File? = null
 
     @Parameter(
-        description = "The path to a Kotlin script to post-process the notice report before writing it to disk.",
-        names = ["--post-processing-script"],
+        description = "The path to a Kotlin script to pre-process the notice report before writing it to disk.",
+        names = ["--pre-processing-script"],
```

Comment:
optional: Consider naming it just `processing-script` so that there is no more interpretation about what `pre` `post` actually mean needed.

Reply:
I prefer to keep the "pre".